wazo_confd/database/email.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from xivo_dao.alchemy.mail import Mail
+from xivo_dao.helpers.db_manager import Session
+
+
+def get():
+    return Session.query(Mail).first()
+
+
+def update(email_config):
+    mail = Session.query(Mail).first()
+    for name, value in email_config.items():
+        setattr(mail, name, value)
+    Session.flush()
```

Comment:
Avoid mixing wazo-confd logic. In confd DAOs and service take an object (sqlalchemy), not a dict. The setattr is done at the http layer.
I also think that the migration from xivo-dao to wazo-confd would be easier if everything is on xivo-dao repository .. but I guess is arguable (i.e. move xivo_dao/resources to wazo-confd, update import and voila)